src/main/Bug/BugReportMessage.java

```diff
@@ -5,7 +5,8 @@
 public enum BugReportMessage {
```

Comment:
make BugReportMessage implement Message and then copy those fields/methods from UserMessage

Reply:
couldn't find Message interface (userMessage doesn't have it either)